### DISCUSSION TITLE
how to use in the unity game engine

### DISCUSSION THREAD

**Discussion by coolcatcoder** [InteractiveComputerGraphics/SPlisHSPlasH#190] [Q&A]
Hey this may be a stupid question, but say i wanted to use this in the unity game engine, would i somehow make it render to unity, or instead simulate the fluid and then send a list of particles and their positions every frame. I reckon using a native plugin for unity would be easiest to get the api into unity btw. Any help appreciated!

**Comment by janbender** [comment]
Hi, 

this is possible. Currently, there are implementations to use the AntTweakBar or imgui in combination with OpenGL as GUI. Analogously, you can implement corresponding classes for unity in /Simulator/GUI and /GUI.

Hope that helps.
